### ISSUE TITLE
Transfer parts from stock to another without merging

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#6951]
### Discussed in https://github.com/inventree/InvenTree/discussions/6825

<div type='discussions-op-text'>

<sup>Originally posted by **MrShippeR** March 23, 2024</sup>
Hello comunity,
I would like ask you how use system right. I have 2 stock locations:

- New items on transport to Central stock
- Central stock

### Idea
My way of using Inventree is: I Add items at PC web interface to "items on transport" and someone transport it with car to "Central stock". 
When parts arrive, I want someone get Android app and check in delivered parts (count, undamaged...). To assign them to shelf where the part belogs, where is QR code for item and some parts are already there. Operator then scan QR on shelf and it should increase count of existing item. I should see that increase in Item history.

### Reality
When someone change location of item from "items on transport" to "Central stock" there are 2 separate items and I must Merge them to one. And when I use Merge button, Item history is out/non complete/starting from blank.

### Question
Is there method how do this without merging? Idealy by scanning QRs?</div>

**Comment by SchrodingersGat** [comment]
Related issue: https://github.com/inventree/inventree-app/issues/480

**Comment by SchrodingersGat** [comment]
@MrShippeR if you would like the development team to assign priority to this you can fund the issue using the link above

**Comment by Velenka1** [comment]
Hi, sorry to bother. I am unclear on the history shown here but will this issue will be addressed? We have just run into this on our system and for us it is a bit silly to have multiple stock items for a single part at the same location.

While merging does work, it is a hassle, especially where the stock history is concerned (which is important for us).

We also observe this problem occurring upon receiving line items from a purchase order.

**Comment by SchrodingersGat** [comment]
@Velenka1 there has been no movement on this - no-body has stepped up to implement this and the core dev team are focused on larger efforts. Would this be something that you would be keen to implement?

**Comment by neilbeukes** [comment]
I'm also facing this issue and would like to get a solution to this. 

I'm willing to pick this up. 

The Idea is to have an additional option here (where the star is) that you can enable and it will "merge" it into the first existing stock it finds of the same part in the new location. 

<img width="2285" height="241" alt="Image" src="https://github.com/user-attachments/assets/57d013e0-5e6e-465b-9174-b13f921288c3" />

Sorry about the mock design quality (insert graphic design is my passion meme here)

**Comment by matmair** [comment]
I am sure we would review a PR; the request is pretty old so it should be verified that this was not implemented already

**Comment by neilbeukes** [comment]
I don't believe it's added yet, it's something I was struggling with today, and I couldn't see any existing option or functionality that allows merging by default on transfer. ( Or auto merging at all, for that matter)
